### PR TITLE
Remove QLI.

### PR DESCRIPTION
**Pull request by asfernandes**

### CODE REVIEW

**Comment by asfernandes** [comment]
Anyone against merge this?

**Comment by dyemanov** [comment]
Merge it.

File: builds/mac_os_x/CS/Helpers.make [hunk 1]
```diff
@@ -262,24 +243,11 @@ $(PS_FW_FLAG): $(FULL_FW_FLAG) $(UPG_FW_FLAG)
 	touch $(PS_FW_FLAG)
 darwin_pseudo_fw_clean:
 
-build_dbs_: $(MSG_DB) $(HELP_DB) $(META_DB)
-build_dbs_clean:
-	rm -f $(MSG_DB) $(HELP_DB) $(META_DB)
-build_dbs_%:
```
Comment (mrotteveel) [inline builds/mac_os_x/CS/Helpers.make:268]: Maybe I'm missing something, but shouldn't this remain for `$(MSG_DB)`?

File: src/msgs/transmsgs.de_DE2.sql [hunk 2]
```diff
@@ -2,33 +2,33 @@ set bulk_insert INSERT INTO TRANSMSGS (ENG_TEXT, ENG_ACTION, ENG_EXPLANATION, FA
 -- Notice LOCALE is hardcoded as 'de_DE' in the INSERT statement and therefore the rows below have one fields less than the table.
 -- JRD
 ('', NULL, NULL, 0, 0, '', NULL, NULL, 'daves', '1993-07-16 12:15:54')
-('arithmetic exception, numeric overflow, or string truncation', NULL, NULL, 0, 1, 'Arithmetischer Fehler, numerischer �berlauf oder Zeichenkette abgeschnitten', NULL, NULL, 'daves', '1993-07-16 12:16:07')
```
Comment (mrotteveel) [inline src/msgs/transmsgs.de_DE2.sql:5]: These changes seem to have corrupted the character encoding of the file.

Comment (asfernandes) [reply src/msgs/transmsgs.de_DE2.sql:5]: Fixed.

File: builds/mac_os_x/CS/Helpers.make (hunk 1 above)
Comment (asfernandes) [reply builds/mac_os_x/CS/Helpers.make:268]: I'm not sure this file should even exist.

**Pull request merged**